ci-operator/step-registry/interop-tests/cnv-tests-e2e-deploy/interop-tests-cnv-tests-e2e-deploy-ref.yaml

```diff
@@ -4,7 +4,7 @@ ref:
     Run containerized CNV e2e-deploy tests
   from: cnv-ci
   commands: interop-tests-cnv-tests-e2e-deploy-commands.sh
-  timeout: 1h0m0s
+  timeout: 6h0m0s
```

Comment:
Why did you bump it so much? this tests should take only 20-30 minutes.

Reply:
reverted this change